tests/Unit/EmogrifierTest.php

```diff
@@ -684,6 +684,92 @@ public function emogrifyNotAppliesCssToNonMatchingElements($css, $expectedHtml)
         static::assertContains(sprintf($expectedHtml, $cssDeclaration1, $cssDeclaration2), $result);
     }
 
+    /**
+     * Data provider for {@see EmogrifierTest::emogrifyAppliesMostSpecificSelectorCssToMatchingElements}
```

Comment:
I don't think we need the description and the `@see` here as the method name is "speaking" enough.

Reply:
Removed